client/pom.xml

```diff
@@ -168,6 +168,11 @@
     </dependency>
     <dependency>
       <groupId>org.apache.cloudstack</groupId>
+      <artifactId>cloud-plugin-network-vsp</artifactId>
```

Comment:
Yes, The dependency can be found here: http://cs.mv.nuagenetworks.net/releases/net/nuagenetworks/vsp/nuage-vsp-acs-client/1.0.0/